### DISCUSSION TITLE
SSO integration

### DISCUSSION THREAD

**Discussion by zhangyang-bytedance** [kubevela/kubevela#6189] [General]
How to integrate kubevela within enterprise's internal SSO system, like Lark ？

**Comment by zhangyang-bytedance** [comment]
For example：
![image](https://github.com/kubevela/kubevela/assets/95331997/5d4a8c92-68ac-450f-8269-b34e1e712b92)
![image](https://github.com/kubevela/kubevela/assets/95331997/147619b3-f9cb-4ec7-8821-850264b8f360)

**Comment by barnettZQG** [comment]
@zhangyang-bytedance This guide maybe could help you: http://kubevela.net/docs/tutorials/sso

**Reply by lucasfcnunes** [reply]
Is there a way to use another IdP besides Dex?
